src/defaults/html.pipe.js

```diff
@@ -57,6 +58,7 @@ const htmlpipe = (cont, context, action) => {
     .every(dump.record)
     .every(validate).when(() => !production())
     .every(timer.update)
+    .before(resolveRef).expose('resolve').when(hascontent)
```

Comment:
Also add `.when(() => !production())` so that we don't have to emulate on localhost.

Reply:
no, if the content repo is non-localhost, I also want fresh content...

Reply:
So then we should check for `localhost`

Reply:
it's done here:
https://github.com/adobe/helix-pipeline/pull/384/files#diff-af7e4527a3d340abcce3d521690db877R35